Makefile

```diff
@@ -20,8 +20,8 @@ export XDG_DATA_HOME ?= /tmp/.local/share
 include .bingo/Variables.mk
 
 # ARTIFACT_PATH is the absolute path to the directory where the operator-controller e2e tests will store the artifacts
-# for example: ARTIFACT_PATH=/tmp/artifacts make test 
-export ARTIFACT_PATH ?= 
+# for example: ARTIFACT_PATH=/tmp/artifacts make test
+export ARTIFACT_PATH ?=
```

Comment:
And yet, Makefiles are one of the few types of files where end-of-line whitespace may be significant!

Reply:
My editor removed trailing spaces automatically here. But I believe it is not significant here.

Reply:
No, it's not... but it's just another Makefile-ism.